timeseries/tests/unittests/models/test_local.py

```diff
@@ -3,6 +3,7 @@
 import pandas as pd
 import pytest
```

Comment:
@tonyhoo Regarding the tests for model performance: Currently, CI for time series does not include any regression tests. The reasoning was the models could be changing quite frequently, and keeping track of the performance ranges for individual models would be tedious. However, I agree that now we should probably be looking to add these tests, as the model set is becoming more stable. Do you think this it's fine if we add these tests after v0.8, or do you think it has higher priority and we should do this asap?

Reply:
We can add these post 0.8 and make sure it is incorporated into the benchmark and dashboard project